src/model/schema/ThunderstoreSchema.ts

```diff
@@ -0,0 +1,131 @@
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
+import schema from "../../../schema.json";
+import schemaSchema from "../../../schemaSchema.json";
+import { GameSelectionDisplayMode } from "../game/GameSelectionDisplayMode";
+import R2Error from "../errors/R2Error";
+
+// Disabled for now, will become relevant when we migrate away from JSON modules.
+export interface Schema {
+    schema_version: string,
+    games: [string:SchemaEntry],
+    communities: [string:ThunderstoreEntry],
+    packageInstallers: [],
+    modloaderPackages: [ModloaderPackage],
+}
+
+export interface SchemaEntry {
+    uuid: string,
+    label: string,
+    meta: EntryMeta,
+    distributions: [EntryDist],
+    thunderstore?: ThunderstoreEntry,
```

Comment:
Based on the schemaSchema, the type here should be `ThunderstoreEntry|null`, but this amounts to `ThunderstoreEntry|undefined`. Probably applies to other interfaces with `?` as well?